Content.Server/DeadSpace/MutilatedCorpse/MutilatedCorpseComponent.cs

```diff
@@ -1,6 +1,4 @@
-using Content.Shared.Damage.Prototypes;
-using Robust.Shared.Prototypes;
-
+// Мёртвый Космос, Licensed under custom terms with restrictions on public hosting and commercial use, full text: https://raw.githubusercontent.com/dead-space-server/space-station-14-fobos/master/LICENSE.TXT
```

Comment:
```suggestion
// Мёртвый Космос, Licensed under custom terms with restrictions on public hosting and commercial use, full text: https://raw.githubusercontent.com/dead-space-server/space-station-14-fobos/master/LICENSE.TXT

```
Добавь пустую строку